torrent_client/algorithms/announcer.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, torrent_info: TorrentInfo, our_peer_id: bytes, server_port: i
 
         self._last_tracker_client = None
         self._more_peers_requested = asyncio.Event()
-        self._task = None  # type: Optional[asyncio.Task]
+        self._task = None  # type: Optional[]
```

Comment:
What's the purpose of this change?

If it's not intended, please revert it.

Reply:
Sorry for waiting that long,
It was not intended, I will revert it.

Reply:
It should be good now.

Reply:
Thanks!